src/react/utils.js

```diff
@@ -669,7 +674,6 @@ export function getProperty(
     binding = object.symbols.get(property);
   }
   if (!binding) {
-    invariant(!object.isPartialObject(), "getProperty used on a partial object with no binding");
```

Comment:
Why?

Reply:
I originally added this invariant in thinking it would never be possible as part of a normal code path, it turns out that we will run into this with normal code though.